src/Dotnet.Script.DependencyModel/Context/DotnetRestorer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Dotnet.Script.DependencyModel.Logging;
+using Dotnet.Script.DependencyModel.Process;
+
+namespace Dotnet.Script.DependencyModel.Context
+{
+    public class DotnetRestorer : IRestorer
+    {
+        private readonly CommandRunner _commandRunner;
+        private readonly Logger _logger;
+
+        public DotnetRestorer(CommandRunner commandRunner, LogFactory logFactory)
+        {
+            _commandRunner = commandRunner;
+            _logger = logFactory.CreateLogger<DotnetRestorer>();
+        }
+
+        public void Restore(string pathToProjectFile)
+        {
+            _logger.Debug($"Restoring {pathToProjectFile} using the dotnet cli.");            
+            _commandRunner.Execute("dotnet", $"restore {pathToProjectFile}");            
```

Comment:
no need to pass in the runtime ID here? also, I think we should quote this path like mentioned in the PR here https://github.com/filipw/dotnet-script/pull/139/files

Reply:
Fixing quotes. Not passing the runtime id actually cuts the project.assets.json file size in half and still contains all the information we need to resolve runtime dependencies. If we run with a runtime id, we get presented with ALL the native/unmanaged dll's from the core framework and we used to load all of the using `LoadLibrary` which is not needed at all.

Reply:
ah good point!